### DISCUSSION TITLE
SNI功能是否正常？

### DISCUSSION THREAD

**Discussion by AlphaBack** [SagerNet/SagerNet#140] [Q&A] [answered]
同样的trojan节点，在v2ray-ng上加上非服务器域名的SNI可以正常使用，在SageNet上加上SNI网络连不止，去掉SNI可以正常连接。
如果可以的话像v2ray-ng一样增加导出完整json配置的功能

**Comment by nekohasekai** [comment, marked as answer]
不支持不安全的连接, 使用证书或固定证书链代替.
